packages/amplication-client/src/Blueprints/BlueprintRelationForm.tsx

```diff
@@ -76,6 +76,18 @@ const BlueprintRelationForm = ({
             <div>
               <ToggleField name="required" label="Required" />
             </div>
+            <div>
+              <ToggleField
+                name="limitSelectionToProject"
+                label="Limit selection to the same project"
+              />
+            </div>
+            {/* <div>
+              <ToggleField
+                name="parentShouldBuildWithChild"
+                label="Regenerate parent code when building the child"
+              />
+            </div> */}
```

Comment:
Why comment out for new code?

Reply:
I added two new fields in the server, and implemented the first now. Will implement the client side of the second one later today or tomorrow